core/src/main/kotlin/org/evomaster/core/search/gene/regex/CharacterClassEscapeRxGene.kt

```diff
@@ -44,9 +43,14 @@ class CharacterClassEscapeRxGene(
     }
 
     override fun standardMutation(randomness: Randomness, apc: AdaptiveParameterControl, allGenes: List<Gene>) {
+        if (value=="") {
+            // if standardMutation was invoked before calling to randomize
+            // then do a randomize first
+            randomize(randomness, forceNewValue = true, allGenes = allGenes)
+        }
```

Comment:
i think this should be removed. if it does happen that standard mutation is called before a randomized, then it is a bug. we should always randomize before an individual can be mutated. so we should rather investigate why it was not the case (ie bug)

Reply:
Indeed. DIsjunctionListRxGene.randomize() can leave un-initialised the inactive disjunction. Due to this, if standardMutation() changes the activeDisjunction, the gene can be left without randomising.